doc/Programming-guide/Key-platform-features/Common-Type-Library/index.md

```diff
@@ -71,80 +76,124 @@ Matching FQN in a different Tenant | N/A | OK | OK
-To open CT details click the row of the CTs list in corresponding CTL window.
-Below is shown an example of **CT details** window for Application CT.
 ![View Application CTL](attach/view_application_ctl.png)
-To view another version of the CT, click on the version drop-down and select the appropriate version.
 
-## Delete CT schema
+To view another version of the CT, select it form the **Version** drop-down list.
+To create a new version of the CT, click **Create new version**.
+
+### Delete CT schemas
+
+To delete a schema:
 
-CT schema can be deleted via [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/deleteCTLSchemaByFqnVersionTenantIdAndApplicationToken) call or from **CT details** window of Administration UI.
-To delete CT schema open **CT details** and click **Delete** button located in the top of the window.
+- Use the [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/deleteCTLSchemaByFqnVersionTenantIdAndApplicationToken) call.
 
-> **NOTE:** The Kaa admin can delete System CT schema.
-The Tenant admin can delete Tenant CT schema.
-The Tenant developer can delete Application CT schema. 
+OR
 
-## CT promotion
+- Open the **Common type details** page and click **Delete**.
+
+>**NOTE:** Kaa administrator can delete system CT schemas.
+>Tenant administrator can delete tenant CT schemas.
+>Tenant developer can delete application CT schemas.
+{:.note}
+
+### Promote CTs
+
+If you want some of your **Application** scope CTs to be available in the **Tenant** scope, you neeed to promote them.
+To do this, you can use the [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/promoteScopeToTenant) call or use the **Promotion** feature:
+
+1. Open the **Administration UI** page, unfold the **Applications** list.
+
+2. In the sub-list of a chosen application, click **Application CTL**.
+
+3. Click the CT to open the **Common type details** page and click **Promote**.
 
-Users are able to promote the CT's scope from APPLICATION to TENANT, provided that there is no other CT with the identical FQN within the given tenant, and that they have a permission to do so.
-Otherwise an appropriate error will be returned.
-Promotion procedure updates scope of CT including all its versions.
-Promotion of CT can be performed in two ways: via [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/promoteScopeToTenant) call or by clicking **Promote** in the **Application CTL** window of Administration UI as shown below.
 ![Promote CTL](attach/application_ctl_promote.png)
 
-## CT schema import and export
+The CT (including all its versions) is now available in the **Tenant** scope.
+
+>**NOTE:** You cannot promote a CT from the **Application** scope if there is a CT in the **Tenant** scope with the same FQN.
```

Comment:
This is inaccurate. Read the original more carefully.

You cannot promote a CT from the **Application** scope if there is another CT within the given Tenant with the same FQN, or the CT in question has dependencies on other CTs in application scope.